src/ophyd_async/epics/adcore/_core_io.py

```diff
@@ -44,9 +44,6 @@ def __init__(self, prefix: str, name: str = "") -> None:
 class NDPluginStatsIO(NDPluginBaseIO):
     """
     Plugin for computing statistics from an image or region of interest within an image.
-    Each boolean signal enables or disables all signals in the appropriate Enum class.
-    The enum signals may used in the ScalarSignals kwargs of a HDFWriter, and are also
-    read-only signals on the plugin.
```

Comment:
Still useful to note which signals are enabled/disabled by the bool signals: maybe sort the signals such that each `compute_` is directly above all of the signals it then enables?